db/schema.rb

```diff
@@ -39,6 +39,15 @@
     t.datetime "updated_at",                 null: false
   end
 
+  create_table "paradigms", force: :cascade do |t|
+    t.string   "name"
+    t.text     "description"
+    t.integer  "ranking"
+    t.boolean  "active"
```

Comment:
active should have default true. It's in the migration file so you might just have to run db:migrate